source/UberRides/RidesUtil.swift

```diff
@@ -34,8 +34,7 @@ import CoreLocation
 }
 
 class ColorUtil {
-    /// Convert hex color code into UIColor
-    static func uberUIColor(color: UberButtonColor) -> UIColor {
+    static func convertHexToUIColor(color: UberButtonColor) -> UIColor {
```

Comment:
This function isn't actually converting hex to UIColor, it's converting an UberButtonColor to UIColor.
A better name would probably be `colorForUberButtonColor`